### PR TITLE
[CX-3647] feat: Release SWA landing page redesign

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Feature**

This PR solves [CX-3647](https://artsyproduct.atlassian.net/browse/CX-3647)

### Description

- Releasing the SWA landing page redesign
- Design fixes

🟥 Do not merge - we need to QA first 🧪 🧪 🧪 

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CX-3647]: https://artsyproduct.atlassian.net/browse/CX-3647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — APPROVED** [review]
LGTM - only comment is about commented code

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx [hunk 1]
```diff
@@ -129,12 +129,12 @@ export const SPECIALISTS: SpecialistsData[] = [
     email: "alan@artsy.net",
   },
   // {
-  //   specialty: "priveteSalesAndAdvisory",
+  //   specialty: "privateSalesAndAdvisory",
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx:132]: They will be added soon when the bios and images are ready. How about this? I'll remove them here from the code and create a Jira ticket for adding them when the bios and images are ready (with the full list attached as a file). Like this we don't have to type them again.

[the remark this answers is not in the document]
Comment (olerichter00) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx]: I have removed them and added the list to https://artsyproduct.atlassian.net/browse/CX-3701

[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx]: sounds great. thanks Ole

**Review by MounirDhahri — APPROVED** [review]
Thanks for addressing the review comments. LGTM

**Pull request merged**